core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -437,6 +470,31 @@ private ExprValue exprAddDateDays(ExprValue date, ExprValue days) {
         : exprValue);
   }
 
+  /**
+   * CONVERT_TZ function implementation for ExprValue.
+   * Returns null for time zones outside of +13:00 and -12:00.
+   *
+   * @param startingDateTime ExprValue of DateTime that is being converted from
+   * @param fromTz ExprValue of time zone offset(string), representing the time to convert from.
+   * @param toTz ExprValue of time zone offset(string), representing the time to convert to.
+   * @return DateTime that has been converted to the to_tz timezone.
+   */
+  private ExprValue exprConvertTZ(ExprValue startingDateTime, ExprValue fromTz, ExprValue toTz) {
+    ZoneId convertedFromTz = ZoneId.of(fromTz.stringValue());
+    ZoneId convertedToTz = ZoneId.of(toTz.stringValue());
+
+    if (!isTimeZoneValid(convertedFromTz)
+          || !isTimeZoneValid(convertedToTz)) {
+      return ExprNullValue.of();
```

Comment:
```mysql> SELECT convert_tz('2010-01-01 10:10:10', '-15:00', '+00:00');
+-------------------------------------------------------+
| convert_tz('2010-01-01 10:10:10', '-15:00', '+00:00') |
+-------------------------------------------------------+
| NULL                                                  |
+-------------------------------------------------------+
1 row in set (0.01 sec)

mysql> SELECT convert_tz('2010-01-01 10:10:10', '-10:00', '+00:00');
+-------------------------------------------------------+
| convert_tz('2010-01-01 10:10:10', '-10:00', '+00:00') |
+-------------------------------------------------------+
| 2010-01-01 20:10:10                                   |
+-------------------------------------------------------+
1 row in set (0.00 sec)``` Here is a run from mysql. Returns null for values outside of range. ` This function returns NULL if any of the arguments are invalid, or if any of them are NULL.` This is also stated in the docs.  I think the comment about no conversion means that it doesn't do anything, because it returns null.